src/main/kotlin/org/wycliffeassociates/otter/common/wav/WavOutputStream.kt

```diff
@@ -0,0 +1,111 @@
+package org.wycliffeassociates.otter.common.wav
+
+import java.io.BufferedOutputStream
+import java.io.Closeable
+import java.io.FileNotFoundException
+import java.io.FileOutputStream
+import java.io.IOException
+import java.io.OutputStream
+import java.io.RandomAccessFile
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+
+/**
+ * Created by sarabiaj on 10/4/2016.
+ */
+class WavOutputStream @Throws(FileNotFoundException::class)
+@JvmOverloads constructor(
+    internal val wav: WavFile,
+    append: Boolean = false,
+    private val buffered: Boolean = false
+) : OutputStream(), Closeable, AutoCloseable {
+
+    private val outputStream: OutputStream
+    private lateinit var bos: BufferedOutputStream
```

Comment:
If you add an `else` clause to its initialization (line 48), you don't need lateinit and this can be 
```kotlin
private val bos: BufferedOutputStream?
```